.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "tests/test262/test262"]
+	path = tests/test262/test262
+	url = https://github.com/tc39/test262
```

Comment:
I prefer to add scripts for checking out submodules to package.json (or similar) to make it easier for people to make use of them, since most people don't do recursive checkouts and not everyone is familiar with the git submodule commands.

It would be nice to have something like that here to make this more accessible to people.